tests/NBomber.Sinks.Timescale.Tests/TimescaleTest.cs

```diff
@@ -100,10 +100,12 @@ public async Task When_Scenario_Finished_The_DataBase_Should_Contain_Data()
             var sessionTableCount = await fixture.TestHelper.GetDataCount(TableNames.SessionsTable);
             var stepStatsTableCount = await fixture.TestHelper.GetDataCount(TableNames.StepStatsTable);
             var metricsTableCount = await fixture.TestHelper.GetDataCount(TableNames.MetricsTable);
+            var isExistHtmlReport = await fixture.TestHelper.IsExistData(TableNames.SessionsTable, ColumnNames.SessionResult + "->>'HtmlReport'");
```

Comment:
you need to pull the value and deserialize it and check that it contains some HTML